docs/cloud/assets.md

```diff
@@ -0,0 +1,15 @@
+# Step 1: Assets
+
+In its installation, Feldera Cloud requires several assets, most notably
+Helm packages and container images. These assets are made available to you
+along with your Feldera license key. They are not publicly available.
+Please contact us at `learnmore@feldera.com` if you would like a trial.
```

Comment:
Shall I fully get rid of "trial" mention, as in reword this to "(...) if you would like to try out Feldera Cloud." ?